gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -326,7 +327,10 @@ private void runJobExecutionLauncher() throws JobException {
         // make sure the planning job is initialized (or visible) to other parallel running threads,
         // so that the same critical section check (querying Helix for job completeness)
         // can be applied.
-        HelixUtils.waitJobInitialization(planningJobHelixManager, newPlanningId, newPlanningId);
+        Duration submissionTimeout = Duration.ofSeconds(PropertiesUtils
```

Comment:
The planning job (i.e. distributed mode) is used in GaaS. This change is usable by GaaS in the event that GaaS also sees similar Helix / ZK bottlenecks. 

The default is the same as the previous 5 minute wait, so it is backwards compatible